GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -1184,7 +1199,12 @@ private void StageFile(string filename)
             void ProcessStart(FormStatus form)
             {
                 form.AddMessageLine(string.Format(_stageFilename.Text, filename));
-                string output = Module.RunGitCmd("add -- \"" + filename + "\"");
+                GitArgumentBuilder args = new GitArgumentBuilder("add")
+                    {
+                        "--",
+                        filename?.QuoteNE()
+                    };
```

Comment:
mis-indented

Reply:
Don't worry about code formatting yet.  Not ready to go through that part of the code.  Although, VS wasn't complaining.

Reply:
I have a stylecop installed in my head, it sets off when I see things 😆

Reply:
LMAO!

Reply:
![image](https://user-images.githubusercontent.com/3629489/46256726-99324f00-c47c-11e8-98c1-335fe4d24688.png)

![image](https://user-images.githubusercontent.com/3629489/46256752-e31b3500-c47c-11e8-9208-f7fa2ebab867.png)

Reply:
This just looks untidy and inconsistent